### PR TITLE
Support Swift Package manager

### PR DESCRIPTION
**Pull request by jjamminjim**

### CODE REVIEW

**Comment by jjamminjim** [comment]
@pjvea  Can you take a look at this PR and approve it? Thanks

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
```
Comment (pjvea) [inline Package.swift:6]: Ohhhh swift package manager support 😻

**Review by pjvea — APPROVED** [review]
Looks gouda! :)

**Comment by jjamminjim** [comment]
@pjvea Any chance you can merge this PR? You can add me as a collaborator if you like.

**Comment by pjvea** [comment]
@jjamminjim I actually don't have permission to update the settings lol. I will merge this and talk to someone about it. :)

**Comment by jjamminjim** [comment]
@pjvea Thanks for the merge man. Can you also submit a release - 1.4.0?

**Comment by pjvea** [comment]
@jjamminjim Done :) 

https://github.com/busybusy/DBC-Apple/releases/tag/1.4.0

**Pull request merged**